apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/McsTransformerIT.java

```diff
@@ -452,15 +547,32 @@ public static McsClaim.Builder createDefaultClaimBuilder() {
           .setIdrHdrToDos(CLM_THRU_DT);
     }
 
+    /**
+     * Static function creates a data parser.
+     *
+     * @param parserData the data to be parsed
+     * @return {@link Parser}
+     */
     public static Parser.Data<String> createDataParser(Map<String, String> parserData) {
       return new Parser.Data<>() {
         private final Map<String, String> dataMap = parserData;
 
+        /**
+         * Gets the {@link long} of the entryNumber.
+         *
+         * @return {@link long}
+         */
         @Override
```

Comment:
This is an override; we can just inheritDoc. (Same for the other overrides)
```suggestion
        /** {@inheritDoc} */
        @Override
```